lib/ProgramCall.js

```diff
@@ -28,6 +28,7 @@ class ProgramCall {
    * Using ``off`` or ``fast`` continues executing the script. The Difference is that ``fast``
    * will log a brief error report and ``off`` will not.
    * @property {string} [func] - The target function of the service program.
+   * @property {string} [mode=opm|ile] - Determines the program call mode.
```

Comment:
Checkout the [preview](https://nodejs-itoolkit--363.org.readthedocs.build/en/363/ProgramCall.html#programCallConfig) here with the doc addition. We should state that the valid values are `opm` and `ile`

```suggestion
   * @property {string} [mode=opm|ile] - Determines the program call mode. Valid values are`opm` and `ile`.
```